src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private String name = "Circle";
+
+    private String color;
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    public Circle() {
+    }
```

Comment:
Avoid providing a default constructor that doesn't set the class fields to meaningful values. If it's not necessary for the class functionality, consider removing it.